scrabbler/simulation.py

```diff
@@ -43,6 +43,12 @@ def simulate(self):
             self.game.show()
 
         self.print_end_game_message()
+       
+        p1_score = self.players[1].get_score()
+        p0_score = self.players[0].get_score()
+        print(p1_score)
+        print(p0_score)
```

Comment:
Can you remove these print statements? Or if you want to keep them because you think they add value, can you add some text so it's clear what's being printed to the console?